values.yaml

```diff
@@ -37,16 +39,62 @@ mysql:
     mysql.cnf: |-
       [client]
       default-character-set = utf8mb4
-      
+
       [mysqld]
       character-set-server = utf8mb4
       collation-server = utf8mb4_bin
       explicit_defaults_for_timestamp = 1
-      
+
+      [mysql]
+      default-character-set = utf8mb4
+
+## MariaDB helm chart
+## https://github.com/bitnami/charts/tree/master/bitnami/mariadb
+mariadb:
+ ## Whether to deploy a mariadb server
+  enabled: false
+  auth:
+    rootPassword: xwiki
+    username: xwiki
+    password: xwiki
+    database: xwiki
+  persistence:
+    size: 20Gi
+  image:
+    tag: 10.5.10
+  primary:
+    configuration: |-
+      [mysqld]
+      skip-name-resolve
+      explicit_defaults_for_timestamp = 1
+      basedir=/opt/bitnami/mariadb
+      plugin_dir=/opt/bitnami/mariadb/plugin
+      port=3306
+      socket=/opt/bitnami/mariadb/tmp/mysql.sock
+      tmpdir=/opt/bitnami/mariadb/tmp
+      max_allowed_packet=16M
+      bind-address=0.0.0.0
+      pid-file=/opt/bitnami/mariadb/tmp/mysqld.pid
+      log-error=/opt/bitnami/mariadb/logs/mysqld.log
+      character-set-server = utf8mb4
+      collation-server = utf8mb4_bin
+
+      [client]
+      port=3306
+      socket=/opt/bitnami/mariadb/tmp/mysql.sock
+      default-character-set = utf8mb4
+      plugin_dir=/opt/bitnami/mariadb/plugin
+
+      [manager]
+      port=3306
+      socket=/opt/bitnami/mariadb/tmp/mysql.sock
+      pid-file=/opt/bitnami/mariadb/tmp/mysqld.pid
+
```

Comment:
This is the diff between your config and the original MariaDB. Is it really necessary?
```diff
 [mysqld]
 skip-name-resolve
-explicit_defaults_for_timestamp
+explicit_defaults_for_timestamp = 1
 basedir=/opt/bitnami/mariadb
 plugin_dir=/opt/bitnami/mariadb/plugin
 port=3306
 socket=/opt/bitnami/mariadb/tmp/mysql.sock
 tmpdir=/opt/bitnami/mariadb/tmp
 max_allowed_packet=16M
 bind-address=0.0.0.0
 pid-file=/opt/bitnami/mariadb/tmp/mysqld.pid
 log-error=/opt/bitnami/mariadb/logs/mysqld.log
-character-set-server=UTF8
-collation-server=utf8_general_ci
+character-set-server = utf8mb4
+collation-server = utf8mb4_bin
 [client]
 port=3306
 socket=/opt/bitnami/mariadb/tmp/mysql.sock
-default-character-set=UTF8
+default-character-set = utf8mb4
 plugin_dir=/opt/bitnami/mariadb/plugin
 [manager]
 port=3306
 socket=/opt/bitnami/mariadb/tmp/mysql.sock
 pid-file=/opt/bitnami/mariadb/tmp/mysqld.pid
```